src/app.ts

```diff
@@ -10,6 +12,14 @@ app.register(cors, {
   origin: ['http://127.0.0.1:5173', 'http://localhost:5173'],
 })
 app.register(userRoutes)
+app.register(authRoutes)
+
+app.register(fastifyJwt, {
+  secret: 'squad40',
```

Comment:
Hey, this secret is what makes us certain that our application is secure. It should be hidden, and only the application should know about it.

Could you create an environment variable to abstract that? It would be great if you could also add it to our .env.example file.